packages/node-opcua-nodeid/test/test_nodeid.js

```diff
@@ -163,21 +163,21 @@ describe("testing coerceNodeId", function() {
         buffer.writeUInt32BE(0xb1dedada, 0);
         buffer.writeUInt32BE(0xb0b0abba, 4);
         const nodeId = coerceNodeId(buffer);
-        nodeId.toString().should.eql("ns=0;b=b1dedadab0b0abba");
-        nodeId.value.toString("hex").should.eql("b1dedadab0b0abba");
+        nodeId.toString().should.eql("ns=0;b=sd7a2rCwq7o=");
+        nodeId.value.toString("base64").should.eql("sd7a2rCwq7o=");
     });
 
     it("should coerce a OPAQUE buffer in a string ( with namespace ) ", function() {
-        const nodeId = coerceNodeId("ns=0;b=b1dedadab0b0abba");
+        const nodeId = coerceNodeId("ns=0;b=dGVzdDEyMzQ1Cg==");
         nodeId.identifierType.should.eql(NodeIdType.BYTESTRING);
-        nodeId.toString().should.eql("ns=0;b=b1dedadab0b0abba");
-        nodeId.value.toString("hex").should.eql("b1dedadab0b0abba");
```

Comment:
Keep the hex form : nodeId.value.toString("hex").should.eql("b1dedadab0b0abba");

Reply:
reverted